activerecord/lib/active_record/relation/query_methods.rb

```diff
@@ -354,17 +354,32 @@ def group!(*args) # :nodoc:
       self
     end
 
-    # Allows to specify an order attribute:
+    # Order can be used to sort the results of a query.
```

Comment:
I believe `Applies an order to the query.` is way too vague. It uses the method name to define itself.

Reply:
> It uses the method name to define itself.

I'm not sure that is a problem.  I see the analogy to a dictionary errantly defining a word in terms of itself, but that's not exactly what we're doing here.  The method name is just a symbol.  It could be `foobar`, and "Applies an order to a query" would still be an apt description.

Would you prefer...?

```suggestion
    # Applies an +ORDER BY+ clause to a query.
```

Reply:
Yeah, I think that explanation offers more to the reader, and I was looking for a way to work the SQL into the description when I first wrote this.